my-burger-app/src/components/news.jsx

```diff
@@ -0,0 +1,15 @@
+const mockData = ['', '', '', ''];
```

Comment:
Better to have something inside

```suggestion
const mockData = ['news1', 'news2', 'news3', 'news3'];
```